python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -116,6 +117,7 @@ def __init__(self, num_processes: int = 1,
     def compile(model: nn.Module,
                 loss: _Loss = None,
                 optimizer: torch.optim.Optimizer = None,
+                scheduler: _LRScheduler = None,
```

Comment:
Why this change? This PR should focus on OpenCV support; you may submit a separate PR for this change if needed.